### PR TITLE
Correct missing pom metadata

### PR DESCRIPTION
**Pull request by OwenCorrigan76**
The pom file was missing metadata such as scm,  license and developer details. This PR will add the relevant information to the pom.xml file.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -2,9 +2,79 @@
...
+    <developerConnection>scm:git:ssh://github.com:strimzi/drain-cleaner.git</developerConnection>
+    <url>https://github.com/strimzi/drain-cleaner</url>
+  </scm>
+
+  <issueManagement>
+    <system>GitHub</system>
+    <url>https://github.com/strimzi/drain-cleaner/issues</url>
+  </issueManagement>
+
+  <developers>
```
Comment (scholzj) [inline pom.xml:33]: This should be updated to contain all maintainers I guess: https://github.com/strimzi/governance/blob/main/MAINTAINERS

File: pom.xml [hunk 2]
```diff
@@ -2,9 +2,79 @@
...
+  <licenses>
+    <license>
+      <name>The Apache License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+    </license>
+  </licenses>
+
+  <name>Strimzi Drain-Cleaner</name>
+  <description>Dran-Cleaner helps with moving the Apache Kafka pods deployed by Strimzi from Kubernetes
+    nodes which are being drained </description>
```
Comment (scholzj) [inline pom.xml:19]:
```suggestion
  <description>Utility to help with draining nodes running Strimzi-managed Apache Kafka or Apache ZooKeeper pods.</description>
```

File: pom.xml [hunk 3]
```diff
@@ -2,9 +2,79 @@
...
   <version>1.2.0-SNAPSHOT</version>
+
+  <licenses>
+    <license>
+      <name>The Apache License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+    </license>
+  </licenses>
+
+  <name>Strimzi Drain-Cleaner</name>
```
Comment (scholzj) [inline pom.xml:17]:
```suggestion
  <name>Strimzi Drain Cleaner</name>
```

**Review by scholzj — COMMENTED** [review]
Some nits, LGTM otherwise. Thanks for the PR.

File: pom.xml (hunk 1 above)
Comment (OwenCorrigan76) [reply pom.xml:33]: @scholzj Thanks for the quick review. I wasn't sure about which developers to add so this is helpful. Thanks

File: pom.xml (hunk 2 above)
Comment (OwenCorrigan76) [reply pom.xml]: refactored

File: pom.xml (hunk 3 above)
Comment (OwenCorrigan76) [reply pom.xml]: Fixed

File: pom.xml [hunk 4]
```diff
@@ -2,9 +2,85 @@
...
+    <developerConnection>scm:git:ssh://github.com:strimzi/drain-cleaner.git</developerConnection>
+    <url>https://github.com/strimzi/drain-cleaner</url>
+  </scm>
+
+  <issueManagement>
+    <system>GitHub</system>
+    <url>https://github.com/strimzi/drain-cleaner/issues</url>
+  </issueManagement>
+
+  <developers>
```
Comment (im-konge) [inline pom.xml:33]: Sam Hawker is missing from the list below

File: pom.xml [hunk 5]
```diff
@@ -2,9 +2,85 @@
...
+  <licenses>
+    <license>
+      <name>The Apache License, Version 2.0</name>
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+    </license>
+  </licenses>
+
+  <name>Strimzi Drain Cleaner</name>
+  <description>Utility to help with draining nodes running Strimzi-managed Apache Kafka
+    or Apache ZooKeeper pods. </description>
```
Comment (im-konge) [inline pom.xml:19]:
```suggestion
  <description>Utility to help with draining nodes running Strimzi-managed Apache Kafka
    or Apache ZooKeeper pods.</description>
```

**Review by scholzj — APPROVED** [review]

File: pom.xml (hunk 4 above)
Comment (OwenCorrigan76) [reply pom.xml:33]: Now added.

**Pull request merged**